### DISCUSSION TITLE
hyperlink interface to another interface

### DISCUSSION THREAD

**Discussion by akhokhlushin** [directus/directus#2793] [Archive]
Hello,

I've got "Directory" database, with three tables :
"main contains Name and Description (of business)
"contact types" is just a list of available contact types
"contacts" contains contacts, (main_id, contact_type_id and data)

So when i browse collection for main table I cannot see, does business has any contacts, and i can't. But i want, maybe using popup window or just hyperlink (button) to contact collection  prefiltered by main_id.

is any ways to do it without deep diving into js/vue ?

**Comment by rijkvanzanten** [comment]
> So when i browse collection for main table I cannot see, does business has any contacts, and i can't.

If you set up a relational field between "main" and "contact types", it will show the related items on the browse page of "main" already 👍

**Reply by rijkvanzanten** [reply]
> maybe using popup window

It actually does this ☝️ 🙂 It says "3 Items" and then when you hover over it it shows them in a popup
